### DISCUSSION TITLE
Please Add the option to add a lake and or ocean.

### DISCUSSION THREAD

**Discussion by MET905** [niels747/2D-Weather-Sandbox#55] [Ideas]
It would be neat to be able to have the option to control the temperature of an ocean and lake as both masses of water do not always have the same temperature. this would be an awesome feature, and would 100% spark off some intense storm systems. :)

**Comment by mesochaser2425** [comment]
yeah, i agree. sometimes i just want to simulate some moutain enviroment with a lake but you know, one lake is higher than the other. and the higher one is colder in real life. But in the game there is only one water temperature for every kind of water placed on the map.

**Reply by MET905** [reply]
Walt I had no idea you liked Meteorology, ITS TIME TO COOK.

**Reply by mesochaser2425** [reply]
yes i am intrested in meteorology jesse

**Comment by niels747** [comment]
Alright, but then I would make the temperature of all water bodies indepently configurable with the Brush

**Reply by MET905** [reply]
That sounds perfect!

**Reply by JustLearninglol** [reply]
That would be amazing! Now I can simulate El Ninos and La Ninas!

**Reply by Meso22** [reply]
I hate La Ninas but that tool sounds cool

**Reply by MET905** [reply]
Doing this would be a great way to simulate lake breezes, i could use a large body of water as the main ocean for moisture and a little lake with a lower temperature to create crazy lake breeze thunderstorms!
